frontend/leads-list/table/lead-services-cell.component.tsx

```diff
@@ -0,0 +1,274 @@
+import React from "react";
+import { useCss } from "kremling";
+import preventiveHealthIcon from "../../../icons/148705-essential-collection/svg/003-caduceus.svg";
+import nutritionIcon from "../../../icons/148705-essential-collection/svg/009-fruit.svg";
+import immigrationIcon from "../../../icons/148705-essential-collection/svg/027-usa-today-2.svg";
+import financeIcon from "../../../icons/148705-essential-collection/svg/038-money-2.svg";
+import workerRightsIcon from "../../../icons/148705-essential-collection/svg/064-hand-holding-up-a-wrench.svg";
+import familySupportIcon from "../../../icons/148705-essential-collection/svg/061-family-house.svg";
+import communityEngagementIcon from "../../../icons/148705-essential-collection/svg/055-megaphone.svg";
+import focusGroupsIcon from "../../../icons/148705-essential-collection/svg/070-lightbulb-idea.svg";
+
+export default function LeadServicesCell(props: LeadServicesCellProps) {
+  const { leadServices } = props;
+
+  const [healthServices, setHealthServices] = React.useState<HealthService[]>(
+    []
+  );
+  const [nutritionServices, setNutritionServices] = React.useState<
+    NutritionService[]
+  >([]);
+  const [immigrationServices, setImmigrationServices] = React.useState<
+    ImmigrationService[]
+  >([]);
+  const [financialServices, setFinancialServices] = React.useState<
+    FinancialService[]
+  >([]);
+  const [workersRightsServices, setWorkersRightsServices] = React.useState<
+    WorkersRightsService[]
+  >([]);
+  const [familySupportServices, setFamilySupportServices] = React.useState<
+    FamilySupportService[]
+  >([]);
+  const [
+    communityEngagementServices,
+    setCommunityEngagementServices
+  ] = React.useState<CommunityEngagementService[]>([]);
+  const [focusGroupServices, setFocusGroupServices] = React.useState<
+    FocusGroupService[]
+  >([]);
+
+  const scope = useCss(css);
+
+  React.useEffect(() => {
+    const filterHealthServices = leadServices.filter(
+      service => service.programId === 1
```

Comment:
`programId` isn't the same across all databases. Locally versus prod won't always be the same and we can't count on it.